data/decks/kokonoe.json

```diff
@@ -31,8 +31,11 @@
 		},
 		{
 			"timing": "on_continuous_boost",
-			"effect_type": "place_buddy_in_any_space",
-			"buddy_name": "Gravitron"
+			"effect_type": "choice",
+			"choice": [
+				{ "effect_type": "place_buddy_in_any_space", "buddy_name": "Gravitron" },
```

Comment:
actually, can you see if adding 'optional' into the effect does what we want? It looks like place_buddy_in_any_space handles this.
I'd prefer if the UI where you can click a space came up immediately but there's also the cancel/pass button.
Just saves the 99% use case of wanting to place gravitron from needing an extra click.

Reply:
You'll have to update all the tests again though, and maybe double add one where they use the pass and the buddy isn't set.